server/back-end/routes/miah.routes.js

```diff
@@ -0,0 +1,9 @@
+module.exports = app => {
+    const User = require("../controllers/miah.controllers.js");
+
+    // Create a new user
+    app.post("/users", User.create);
+};
+
+require("./miah.routes.js")(app);
+app.listen();
```

Comment:
app.listen() should not be on routes.js as you're already listening on server.js